plugins/woocommerce/tests/e2e-pw/tests/basic/page-loads.spec.js

```diff
@@ -216,35 +223,59 @@ for ( const currentPage of wcPages ) {
 			} )
 			.then( ( r ) => {
 				orderId = r.data.id;
+			} )
+			.catch( ( e ) => {
+				console.error(
+					`Failed to create order ${
+						e.data ? JSON.stringify( e.data ) : ''
+					}`
+				);
+				throw e;
 			} );
-
-		// create customer
-		await restApi
-			.post( `${ WC_API_PATH }/customers`, customer )
-			.then( ( r ) => ( customer.id = r.data.id ) );
```

Comment:
Out of curiosity, were we creating a customer that wasn't used in the test?

Reply:
Yes.